compiler/src/dmd/dclass.d

```diff
@@ -409,19 +409,6 @@ extern (C++) class ClassDeclaration : AggregateDeclaration
         return cd;
     }
 
-    override Scope* newScope(Scope* sc)
```

Comment:
also need to update `aggregate.h` to remove `virtual Scope *newScope(Scope *sc);` and `Scope *newScope(Scope *sc) override;`

Reply:
Noted

Reply:
In `dsymbol.h` should it be updated with this - `Scope* newScope(AggregateDeclaration d, Scope* sc) `and `Scope* newScope(AttribDeclaration d, Scope* sc)` ?

Reply:
No, they are not used by LDC of GDC.

Reply:
Encountered this error 
`dmd/compiler/src/tests/cxxfrontend.cc:236: void test_visitors(): Assertion cd->vtblOffset() == 1' failed.`

This is the line `assert(cd->vtblOffset() == 1);`

refactored into a free function call to this `assert(dmd::vtblOffset(cd) == 1);`

got this  error again `‘vtblOffset’ is not a member of ‘dmd’`

Reply:
to fix that you would need to add it to the corresponding header in https://github.com/dlang/dmd/blob/master/compiler/src/dmd/aggregate.h#L44

Reply:
the cxx test failed ‘`vtblOffset’ is not a member of ‘dmd’`

Reply:
did it like this @thewilsonator 
```
namespace dmd
{
    FuncDeclaration *search_toString(StructDeclaration *sd);
    void semanticTypeInfoMembers(StructDeclaration *sd);
    bool fill(StructDeclaration* sd, const Loc &loc, Expressions &elements, bool ctorinit);
    Scope* newScope(AggregateDeclaration *d, Scope* sc);
    Scope* newScope(AttribDeclaration *d, Scope* sc);
```